inst/templates/BS3/head.html

```diff
@@ -110,7 +110,13 @@
   gtag('config', '{{.}}');
 </script>
 {{/ganalytics}}{{/yaml}}
-
+{{#has_trailingslash}}
+<script>
+  if (!/(\/|html?)$/.test(window.location.href)){
+    window.location.replace(window.location.href + '/');
```

Comment:
Add commenting linking to https://stackoverflow.com/questions/1865837/whats-the-difference-between-window-location-and-window-location-replace — we use `.replace()` because we don't want the old url to appear in the browser history.